procedures/katello/release.md.erb

```diff
@@ -6,19 +6,25 @@
 - [ ] Request Virt Who Configure release from maintainers
 <% unless is_rc %>- [ ] Change Redmine version <%= full_version %> state to Closed
 <% end %>
-- [ ] Clone [tool_belt](https://github.com/theforeman/tool_belt) and run:
+- [ ] Do Cherry-picks: Clone [tool_belt](https://github.com/theforeman/tool_belt) and run:
   - [ ] `./tools.rb setup-environment configs/katello/<%= short_version %>.yaml`
   - [ ] `./tools.rb cherry-picks --version <%= full_version %> configs/katello/<%= short_version %>.yaml`
-  - [ ] Verify tickets in the cherry_picks_<%= full_version %> file are accounted for or additional cherry pick them
-  - [ ] Check for outdated deprecation warnings in the current and next release with `./tools check-deprecation-warnings configs/katello/<%= short_version %>.yaml`. Follow the instructions in the output of the command.
+  - [ ] Open a PR in Katello release branch.  Make sure the PR name starts with [CP] to prevent our automations from adding it to Redmine issues.
+  - [ ] Using `git cherry-pick -x` as needed, verify tickets in the cherry_picks_<%= full_version %> file are accounted for, or additionally cherry-pick them.
+  - [ ] For any cherry-picks that are not needed (including Redmine trackers) you can add them to the `:ignores:` section of `tool_belt` in `configs/katello/<%= short_version %>.yaml`
+- [ ] Check for outdated deprecation warnings in the current and next release with `./tools check-deprecation-warnings configs/katello/<%= short_version %>.yaml`. Follow the instructions in the output of the command. Don't forget to create any Redmine issues needed!
 
-- [ ] In katello release branch:
-  - [ ] Update `lib/katello/version.rb` to <%= full_version %>
+- [ ] Bump version:
+  - [ ] Open a PR (or use cherry-pick PR) against the release branch which updates `lib/katello/version.rb` to <%= full_version %>
   - [ ] Commit: `git commit -m "Release <%= full_version %>"`
+- [ ] In Katello release branch (the real one, not your fork), once PR is merged:
   - [ ] Tag: `git tag -s -m "Release <%= full_version %>" <%= full_version %>`
-  - [ ] Push: `git push --follow-tags`
+  - [ ] Push: `git push upstream --follow-tags` (Must be pushed directly to the release branch, as pull request merges will not preserve tags.)
   - [ ] Generate source gem: `gem build katello.gemspec`
+  - [ ] Ensure you have a working login and password at rubygems.org
   - [ ] Push gem: `gem push katello-<%= full_version %>.gem`
+- [ ] Email the delivery team informing them that the gem is published
+- [ ] Confirm response that the build succeeded (or if necessary, do more cherry-picks and version bumps repeating the steps above)
```

Comment:
This is odd, because the process is a linear document. You here have "the build succeeds" but the build is only done "Once release is out", right? So this should be moved to just before `Post a release announcement`.